muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/organizationmanagement/rest/OrganizationUserManagementRESTService.java

```diff
@@ -171,6 +172,69 @@ public Response searchStaffMembers(
     return Response.ok(responseStaffMembers).build();
   }
   
+  @GET
+  @Path("/studentsSummary")
+  @RESTPermit(OrganizationManagementPermissions.ORGANIZATION_VIEW)
+  public Response studentsSummary(
+      @QueryParam("firstResult") @DefaultValue("0") Integer firstResult,
+      @QueryParam("maxResults") @DefaultValue("10") Integer maxResults) {
+    
+    if (!sessionController.isLoggedIn()) {
+      return Response.status(Status.FORBIDDEN).build();
+    }
+
+    SearchProvider searchProvider = searchProviderInstance.get();
+    if (searchProvider == null) {
+      return Response.status(Status.INTERNAL_SERVER_ERROR).build();
+    }
+
+    UserSchoolDataIdentifier userSchoolDataIdentifier = userSchoolDataIdentifierController
+        .findUserSchoolDataIdentifierBySchoolDataIdentifier(sessionController.getLoggedUser());
+    OrganizationEntity organization = userSchoolDataIdentifier.getOrganization();
+
+    SearchResult result = searchProvider.searchUsers(
+        Arrays.asList(organization),
+        "",
+        null,                                                 // fields
+        Arrays.asList(EnvironmentRoleArchetype.STUDENT),
+        null,                                                 // userGroupFilters
+        null,                                                 // workspaceFilters
+        null,                                                 // userIdentifiers
+        true,                                                 // includeInactiveStudents
+        true,                                                 // includeHidden
+        true,                                                 // onlyDefaultUsers
+        firstResult,
+        maxResults);
+
+    List<Map<String, Object>> results = result.getResults();
+    OrganizationStudentsActivityRESTModel studentActivityRESTModel = null;
```

Comment:
Might as well instantiate the REST object here so that by the end of the method, there's definitely an object to return, even if the search results would be null or empty.